semantic_segmentation/code/dataset.py

```diff
@@ -0,0 +1,257 @@
+from torch.utils import data

```

Comment:
Just, change the path. Refactor this to `/common/datasets/data/datasets.py`

Reply:
let not worry about common for now, once we refactor classifier code as well then we can think of merging the two, I think it's fine even if we keep both fo them separate. Its easier to read and find as well.